CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Added clarification about how collection-level assset object properties do not remove the need for item-level asset object properties in the `item-assets` extension ([#880](https://github.com/radiantearth/stac-spec/pull/880))
+- Added additional acquisition parameters in the `sat` extension
```

Comment:
Better specify explicitly which fields have been added. That helps devs to track down changes.

Reply:
fields listed in changelog